documentation/tutorial.md

```diff
@@ -109,6 +109,14 @@ some functions:
 3. parse_item(): function which will be called to handle response
    downloaded by Crawly. It must return the `Crawly.ParsedItem` structure.
 
+As it is, the init function returns an empty array for `start_urls` which we will
+fix now. If you already know what urls, you want to start crawling, you can place them in
+the array, else you can pick the urls you want to start with from a particular page
+on your website of interest.
+
```

Comment:
> The `init/0` function returns initialization parameters, with `:start_urls` set to an empty list by default. Let's update that to the url that we want our spider to begin crawling from. In this case, we set it to the homepage of `books.toscrape.com`.

Reply:
Since it is a tutorial, we want to avoid readers from deviating from the example, so that they get expected results. Also, the concept of array slightly different for elixir, so better to stick with the correct terminology to avoid confusion.

Reply:
Done